### PR TITLE
Fix 1deg_jra55_iaf configuration

### PR DESCRIPTION
**Pull request by aidanheerdegen**
Lots of issues with metadata and other problems. 

Upgrade to [latest version of ACCESS-OM2](https://github.com/ACCESS-NRI/ACCESS-OM2/releases/tag/2024.03.0).

Need to run the QA checks.

### CODE REVIEW

**Comment by aidanheerdegen** [comment]
Aaargghh. Finally managed to choose the right base on the third try. Hoist by my own petard indeed.

**Comment by dougiesquire** [comment]
FYI, [this](https://github.com/ACCESS-NRI/access-om2-configs/blob/457a9684e67ca711dbe44c432296317f242e5628/config.yaml#L12) line should be 1deg_jra55_iaf

File: metadata.yaml [hunk 1]
```diff
@@ -10,21 +10,26 @@ description: |-
...
+- interannual
+- global
+realm:
+- ocean
+- seaIce
+nominal_resolution: 100 km
+reference: https://doi.org/10.5194/gmd-13-401-2020
+license: CC-BY-4.0
+model: access-om2
+version: 1.0
```
Comment (dougiesquire) [inline metadata.yaml:34]: Should we be setting this on the other releases also?

**Review by dougiesquire — APPROVED** [review]
LGTM

File: metadata.yaml (hunk 1 above)
Comment (aidanheerdegen) [reply metadata.yaml:34]: I was trying to get around some logic errors that @CodeGat has now fixed in order to check later stages of the CI.

The intention of the design was that devs wouldn't need to set the version, there are tools to do this, but these initial cases found some logic bugs/omissions.

So short answer, no, it shouldn't be required.

**Pull request merged**